### DISCUSSION TITLE
getting error in the code which is mentioned in documentation, even all the dependencies are fine

### DISCUSSION THREAD

**Discussion by SakshamSoni-invizAI** [johnbumgarner/wordhoard#22] [Q&A]
`from wordhoard import Homophones

homophone = Homophones(search_string=['horse'])
homophone_results = homophone.find_homophones()
print(homophone_results)`

`---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[15], line 1
----> 1 from wordhoard import Homophones
      3 homophone = Homophones(search_string=['horse'])
      4 homophone_results = homophone.find_homophones()

File ~/.local/lib/python3.8/site-packages/wordhoard/__init__.py:13
     10 __url_documentation__ = 'https://wordhoard.readthedocs.io/en/latest/'
     11 __url_source_code__ = 'https://github.com/johnbumgarner/wordhoard'
---> 13 from .antonyms import Antonyms
     14 from .synonyms import Synonyms
     15 from .hyponyms import Hyponyms

File ~/.local/lib/python3.8/site-packages/wordhoard/antonyms.py:48
     46 from concurrent.futures import ThreadPoolExecutor, as_completed
     47 from typing import Dict, List, Optional, Set, Sized, Tuple, Union
---> 48 from wordhoard.utilities import caching, cleansing, word_verification
     49 from wordhoard.utilities.cloudflare_checker import CloudflareVerification
     51 logger = logging.getLogger(__name__)

File ~/.local/lib/python3.8/site-packages/wordhoard/utilities/caching.py:94
     87         temporary_dict_definition[word] = {pos_category: definitions}
     91 ##################################################################################
     92 # in memory temporary cache for hypernyms
     93 ##################################################################################
---> 94 temporary_dict_hypernyms: Dict[str, list[str]] = {}
     97 def cache_hypernyms(word: str) -> Tuple[bool, Optional[List[str]]]:
     98     try:

TypeError: 'type' object is not subscriptable
`

**Comment by johnbumgarner** [comment]
I'm not sure why you are getting that error.  Here is the some code on my system running `Python 3.9`:

```
from wordhoard import Homophones
homophones = Homophones(search_string='horse')
results = homophones.find_homophones()
print(results)
```

Which produces: 

>  ['horse is a homophone of hoarse']


Are you using the latest version of `Wordhoard`?
